examples/cpp-grpc/proto/example/v1/greeter.proto

```
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package example.v1;
+
+import "google/protobuf/timestamp.proto";
+
+// The greeting service definition
+service GreeterService {
+  // Sends a greeting
+  rpc SayHello(HelloRequest) returns (HelloResponse);
```

Buf (1.54.0)

"example.v1.HelloRequest" is used as the request or response type for multiple RPCs.

(RPC_REQUEST_RESPONSE_UNIQUE)